### PR TITLE
[IMP] point_of_sale: redesign

### PR DESCRIPTION
**Pull request by xlu-odoo**
Redesign and revamp of the point of sale app

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by xlu-odoo** [comment]
@stefanorigano

File: addons/point_of_sale/__manifest__.py [hunk 1]
```diff
@@ -103,12 +103,22 @@
             ('remove', 'web/static/src/core/errors/error_handlers.js'),
             ('remove', 'web/static/src/legacy/legacy_rpc_error_handler.js'),
         ],
+
```
Comment (stefanorigano) [inline addons/point_of_sale/__manifest__.py:106]: unnecessary diff

File: addons/point_of_sale/__manifest__.py [hunk 2]
```diff
@@ -103,12 +103,22 @@
...
+            'web/static/src/legacy/scss/bootstrap_overridden.scss',
+            ('include', 'web._assets_bootstrap'),
+            'web/static/src/**/**/*.variables.scss',
+            'web/static/src/webclient/webclient.scss',
+
             'web/static/fonts/fonts.scss',
             'web/static/src/libs/fontawesome/css/font-awesome.css',
             'web/static/lib/daterangepicker/daterangepicker.css',
             'point_of_sale/static/src/css/pos.css',
+            'point_of_sale/static/src/scss/pos.scss',
```
Comment (stefanorigano) [inline addons/point_of_sale/__manifest__.py:121]: I'd go for a main `pos.variables.scss` ( and eventually `pos.functions.scss`, `pos.mixins.scss`...).
The rest of the SCSS should be split in different files organised by component.. .what do you think ?

**Review by stefanorigano — COMMENTED** [review]
preliminary notes

File: addons/point_of_sale/static/src/xml/Chrome.xml [hunk 3]
```diff
@@ -4,19 +4,21 @@
     <t t-name="Chrome" owl="1">
         <div class="pos" t-att-class="{ 'big-scrollbars': state.hasBigScrollBars }">
             <div class="pos-receipt-print"></div>
-            <div class="pos-topheader" t-att-class="{ oe_hidden: state.uiState !== 'READY' }">
+            <div class="pos-topheader position-absolute start-0 top-0 d-flex w-100 m-0 p-0 fs-5 text-gray" t-att-class="{ oe_hidden: state.uiState !== 'READY' }">
```
Comment (pimodoo) [inline addons/point_of_sale/static/src/xml/Chrome.xml:7]: Hello, as we are refactoring the css, couldn't we just put the flex rules in the scss file corresponding to keep clear xml files and not add noize with class only there for style?

Comment (pimodoo) [reply addons/point_of_sale/static/src/xml/Chrome.xml:7]: I am not sure position-absolute is needed

**Comment by legalsylvain** [comment]
Any chances to see this PR merged in 16 release ?

**Comment by pimodoo** [comment]
> Any chances to see this PR merged in 16 release ?

We made changes in this PR: https://github.com/odoo/odoo/pull/99033

**Comment by legalsylvain** [comment]
Thanks !
